tests/test_redshift.py

```diff
@@ -164,7 +165,12 @@ def test_copy_unload(path, redshift_table, redshift_con, databases_parameters):
         ensure_data_types(df=chunk, has_list=False)
 
 
-def test_copy_upsert(path, redshift_table, redshift_con, databases_parameters):
+def generic_test_copy_upsert(
+    path: str,
+    redshift_table: str,
+    redshift_con: redshift_connector.Connection,
+    databases_parameters: Dict[str, Any],
+) -> None:
```

Comment:
Because the argument is a pytest fixture. I could only make the pytest fixture parameterized, in which case every argument where we need a Redshift table is needed would be executed twice. I couldn't find a feature that says "parameterize this test with two possible fixtures".

Reply:
Gotcha, thanks